pages/owner.units.page.ts

```diff
@@ -0,0 +1,66 @@
+import { Page as PlaywrightPage, expect } from '@playwright/test';
+import Page from './page';
+
+class OwnerUnitsPage extends Page {
+
+    constructor(page: PlaywrightPage) {
+        super(page);
+    }
+
+    firstUnitName = this.page.locator('[class*="OwnerUnitCard_name"]').first();
```

Comment:
Write me abut this

Reply:
I get the name of the first unit on the page